### PR TITLE
Added README/CONTRIBUTING/DEVELOPING docs

### PR DESCRIPTION
**Pull request by ahelwer**
Adapting docs from the TLA+ tools to this repository.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+These tools have a large number existing issues to fix, which you can see on the repo's [issues tab](https://github.com/tlaplus/tlapm/issues).
+
+Non-code contributions are particularly desired!
+TLAPM documentation needs a lot of work.
+The docs are hosted at https://proofs.tlapl.us and live in this repo in the [doc/web](doc/web) directory.
+
+For new features, there are a number of [substantial improvements we'd like to see implemented](todo.txt).
+If you have an idea for a new feature, open an issue in this repository to start a discussion.
+For more fundamental proposals that involve changes to the TLA⁺ language itself, you can [open a RFC](https://github.com/tlaplus/rfcs/issues).
```
Comment (muenchnerkindl) [inline CONTRIBUTING.md:61]: May I suggest writing this in a way that conveys the idea that changes to the TLA+ language should not be necessary, except on very rare and justified occasions?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Often it will be necessary to build community support for your idea; for this you can use the [mailing list](https://groups.google.com/g/tlaplus) and especially the [monthly online community meeting](https://groups.google.com/g/tlaplus/c/CpAEnrf-DHQ/m/YrORpIfSBwAJ).
+Expressions of goodwill like volunteering labor to review pending PRs is also appreciated!
+
+Getting Started
+---------------
+
+Take the time to set up your local development environment.
+See [DEVELOPING.md](DEVELOPING.md) for details.
+Good luck, and have fun!
```
Comment (muenchnerkindl) [inline CONTRIBUTING.md:70]: Why "Good luck", is this standard in CONTRIBUTING? Perhaps "Have fun, and keep in touch!"?

**Review by muenchnerkindl — APPROVED** [review]
Thanks for adding these files, they look good to me. I have two minor comments, but I see that these are directly taken from CONTRIBUTING in tlaplus/tlaplus, so feel free to ignore them.

**Comment by lemmy** [comment]
> Thanks for adding these files, they look good to me. I have two minor comments, but I see that these are directly taken from CONTRIBUTING in tlaplus/tlaplus, so feel free to ignore them.

The version in tlaplus/tlaplus are not set in stone.

**Pull request merged**